contrib/go-chi/chi/chitrace_test.go

```diff
@@ -0,0 +1,87 @@
+package chi
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/DataDog/dd-trace-go/tracer/tracertest"
+	"github.com/go-chi/chi"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTracingMiddleWare(t *testing.T) {
+	assert := assert.New(t)
+
+	tracer, transport := tracertest.GetTestTracer()
+
+	// Router setup
+	r := chi.NewRouter()
+
+	middleware := Middleware("my-service", "/test/{id}", tracer)
+	r.With(middleware).Get("/test/{id}", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(200)
```

Comment:
Why do we need to do this? This should be 200 by default. If it's not, we have a problem.